.golangci.yml

```diff
@@ -70,36 +61,34 @@ linters:
 
   # enable a specific set of linters to run
   enable:
-    - bodyclose
-    - deadcode # enabled by default
-    - dupl
-    - errcheck # enabled by default
-    - funlen
-    - goconst
-    - gocyclo
-    - godot
-    - gofmt
-    - goimports
-    - golint
-    - goprintffuncname
-    - gosec
-    - gosimple # enabled by default
-    - govet # enabled by default
-    - ineffassign # enabled by default
-    - lll
-    - maligned
-    - misspell
-    - nakedret
-    - nolintlint
-    - staticcheck # enabled by default
-    - structcheck # enabled by default
-    - stylecheck
-    - typecheck # enabled by default
-    - unconvert
-    - unparam
-    - unused # enabled by default
-    - varcheck # enabled by default
-    - whitespace
+    - bodyclose           # checks whether HTTP response body is closed successfully
+    - deadcode            # finds unused code
+    - dupl                # code clone detection
+    - errcheck            # checks for unchecked errors
+    - funlen              # detects long functions
+    - goconst             # finds repeated strings that could be replaced by a constant
+    - gocyclo             # computes and checks the cyclomatic complexity of functions
+    - godot               # checks if comments end in a period
```

Comment:
not sure how others feel, but i'm personally in favor of removing this one. sometimes it makes you throw a period in odd places

Reply:
I honestly like `godot` I think it does a really good job at ensure consistency with some of the docs. If we didn't have I'm willing to bet we'd have some sentences with periods and some without in the godocs.

Reply:
k, i mostly don't like it when it forces one when it doesn't make sense, otherwise it's fine

Reply:
for example:

https://github.com/go-vela/server/blob/09c50c25579da62c620ca3c44789baa8f7f42fb5/router/build.go#L44

Reply:
Yeah that's super awkward but I'm guessing the parser is pretty stupid in that case and just looks at the whole comment block. 😢